index.js

```diff
@@ -44,10 +44,12 @@ module.exports = function configParser(yaml, templateFactory) {
         .then(phaseGeneratePermutations)
         // Output in the right format
         .then(doc => ({
+            annotations: Hoek.reach(doc, 'annotations', { default: {} }),
```

Comment:
@r3rastogi The config-parser only parsers the yaml.

For jobs, it goes through the parser, then store the environment, settings, etc. under `permutations`. And because`permutations` is just an object, we can just add additional fields without adding a new column in the database. Example: https://api.screwdriver.cd/v4/jobs/1

For pipeline, we need to add a new field. Sorry I remember telling you we don't need to. 

Basically, you don't need to modify the jobs table, but you need to modify the pipeline table. And since sequelize 4.x doesn't work yet, we can't add a new field. I think it's best to not implement pipeline annotations for now. And when we get sequelize to work, then we can add it.